### PR TITLE
Updated Italian.yml

### PR DESCRIPTION
**Pull request by mattymatty97**
updated Italian language file to fit new API events

### CODE REVIEW

File: SCPDiscordPlugin/Languages/italian.yml [hunk 1]
```diff
@@ -671,7 +713,217 @@ player:
...
+        # starttime - float
+        # timeleft - float
+        # ipaddress - string
+        # name - string
+        # playerid - int
+        # steamid - string
+        # class - Role
+        # team - Team
+        message: "'<var:name>' (<var:class>) ha aperto un generatore nella stanza <var:room>."
+        regex: []
```
Comment (KarlOfDuty) [inline SCPDiscordPlugin/Languages/italian.yml:731]: All messages containing rooms need the extra regex from english.yml to parse the room names.

File: SCPDiscordPlugin/Languages/italian.yml [hunk 2]
```diff
@@ -671,7 +713,217 @@ player:
...
+        # starttime - float
+        # timeleft - float
+        # ipaddress - string
+        # name - string
+        # playerid - int
+        # steamid - string
+        # class - Role
+        # team - Team
+        closed:
+            message: "'<var:name>' (<var:class>) ha chiuso un generatore nella stanza <var:room>."
```
Comment (KarlOfDuty) [inline SCPDiscordPlugin/Languages/italian.yml:747]: -

File: SCPDiscordPlugin/Languages/italian.yml [hunk 3]
```diff
@@ -671,7 +713,217 @@ player:
...
+            regex: [ ]
+        opened:
+            message: "'<var:name>' (<var:class>) ha aperto un generatore nella stanza <var:room>."
```
Comment (KarlOfDuty) [inline SCPDiscordPlugin/Languages/italian.yml:750]: -

File: SCPDiscordPlugin/Languages/italian.yml [hunk 4]
```diff
@@ -671,7 +713,217 @@ player:
...
+        # starttime - float
+        # timeleft - float
+        # ipaddress - string
+        # name - string
+        # playerid - int
+        # steamid - string
+        # class - Role
+        # team - Team
+        message: "'<var:name>' (<var:class>) ha attivato un generatore nella stanza <var:room>."
+        regex: [ ]
```
Comment (KarlOfDuty) [inline SCPDiscordPlugin/Languages/italian.yml:767]: -

File: SCPDiscordPlugin/Languages/italian.yml [hunk 5]
```diff
@@ -671,7 +713,217 @@ player:
...
+        # starttime - float
+        # timeleft - float
+        # ipaddress - string
+        # name - string
+        # playerid - int
+        # steamid - string
+        # class - Role
+        # team - Team
+        message: "'<var:name>' (<var:class>) ha disattivato un generatore nella stanza <var:room>."
+        regex: []
```
Comment (KarlOfDuty) [inline SCPDiscordPlugin/Languages/italian.yml:783]: -

File: SCPDiscordPlugin/Languages/italian.yml [hunk 6]
```diff
@@ -671,7 +713,217 @@ player:
...
+        # room - RoomType
+        # apdrain - float
+        # ipaddress - string
+        # name - string
+        # playerid - int
+        # steamid - string
+        # class - Role
+        # team - Team
+        message: "'<var:name>' (<var:class>) sta parlando in <var:room>."
+        regex: [ ]
```
Comment (KarlOfDuty) [inline SCPDiscordPlugin/Languages/italian.yml:892]: -

File: SCPDiscordPlugin/Languages/italian.yml [hunk 7]
```diff
@@ -671,7 +713,217 @@ player:
...
+        # room - RoomType
+        # apdrain - float
+        # ipaddress - string
+        # name - string
+        # playerid - int
+        # steamid - string
+        # class - Role
+        # team - Team
+        message: "'<var:name>' (<var:class>) ha smesso di parlare da <var:room>."
+        regex: []
```
Comment (KarlOfDuty) [inline SCPDiscordPlugin/Languages/italian.yml:903]: -

File: SCPDiscordPlugin/Languages/italian.yml [hunk 8]
```diff
@@ -671,7 +713,217 @@ player:
...
+        # room - RoomType
+        # apdrain - float
+        # ipaddress - string
+        # name - string
+        # playerid - int
+        # steamid - string
+        # class - Role
+        # team - Team
+        message: "'<var:name>' (<var:class>) ha bloccato tutta la stanza <var:room>."
+        regex: [ ]
```
Comment (KarlOfDuty) [inline SCPDiscordPlugin/Languages/italian.yml:914]: -

File: SCPDiscordPlugin/Languages/italian.yml (hunk 1 above)
Comment (mattymatty97) [reply SCPDiscordPlugin/Languages/italian.yml:731]: why? room names are pretty self explanatory
i've no need to add `containment` translation
or i'll need to make a bigger regex :sweat_smile:

Comment (mattymatty97) [reply SCPDiscordPlugin/Languages/italian.yml:731]: like:
`"nuke":"the nuke silo"`
should become
`"nella stanza nuke":"nel silo della Nuke"`
ecc...

Comment (KarlOfDuty) [reply SCPDiscordPlugin/Languages/italian.yml:731]: Fair enough, your choice.

**Pull request merged**